edx_proctoring/api.py

```diff
@@ -1769,6 +1768,7 @@ def _get_proctored_exam_view(exam, context, exam_id, user_id, course_id):
         student_view_template = 'proctored_exam/ready_to_submit.html'
 
     if student_view_template:
+        log.info("Proctoring context: %s", context)
```

Comment:
I added both of these to debug a sandbox issue. I think it is too verbose, so I'm removing them now.